.github/workflows/build-docker-cache-artifacts.yml

```diff
@@ -0,0 +1,67 @@
+name: Docker Build Cache Save Artifact
+
+on:
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - '**.md**'
+    branches: [main]
+
+jobs:
+  build-and-push-dev-aws:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3
+      - name: Get tag
+        run: echo "RELEASE_VERSION=${GITHUB_REF:10}" >> $GITHUB_ENV
```

Comment:
Question: So this pulls in the most recent released version for testing? Or does this pull in what is latest on main for testing?